main.go

```diff
@@ -127,9 +127,10 @@ func main() {
 		err = cmd.Execute(command, title, assumeYes, maxWait, filename)
 
 	case "play":
+		maxWait := uintArg(args, "--max-wait", cfg.RecordMaxWait())
```

Comment:
I'm not sure if it's good to use `cfg.RecordMaxWait()` here as a default. `RecordMaxWait` is a setting for recording, maybe it would be better to have separate one for playback, like `PlayMaxWait`? Given that you can replay other people's files I can imagine a scenario where I want to use `maxWait = 2` for all playbacks but don't use it for my recordings. Or vice versa. It's hard to imagine what would be the usage patterns here, but it's important to note that `maxWait` applied to recording is a "destructive" operation (it happens once and its effect is persisted) and applied to playback is "non destructive".

What do you think? Any other ideas?